jobs/datadog-nats/templates/process.yaml.erb

```diff
@@ -0,0 +1,9 @@
+init_config:
+
+instances:
+  - name: nats
+    search_string: ['gnatsd']
+
+  - name: nats_stream_forwarder
+    search_string: ['ruby', 'nats_stream_forwarder.rb']
```

Comment:
I think this is incorrect; it will search for  processes with `ruby` *or* `nats_stream_forwarder.rb` in the name, so it'll assume this one-liner `ruby -e 'sleep'`  is a nats_stream_forwarder.

https://github.com/DataDog/dd-agent/blob/master/conf.d/process.yaml.example#L15-L16